infra-as-code/bicep/modules/policy/assignments/alzDefaults/alzDefaultPolicyAssignments.bicep

```diff
@@ -505,6 +505,13 @@ var varPrivateDnsZonesFinalResourceIds = {
   azureCognitiveSearchPrivateDnsZoneId: '${varPrivateDnsZonesBaseResourceId}privatelink.search.windows.net'
 }
 
+var varPolicyAssignmentScopeName = '${parTopLevelManagementGroupPrefix}${parTopLevelManagementGroupSuffix}'
+var varPolicyExemptionConfidentialOnlineManagementGroup = '${parTopLevelManagementGroupPrefix}-landingzones-confidential-online${parTopLevelManagementGroupSuffix}'
```

Comment:
Can we use the existing variables within this module like varManagementGroupIds.landingZonesConfidentialCorp and varManagementGroupIds.landingZonesConfidentialOnline to keep it dry?